src/components/BookList.js

```diff
@@ -1,34 +1,55 @@
-import React from 'react';
+import React, { useState } from 'react';
 import PropTypes from 'prop-types';
 import { useDispatch } from 'react-redux';
 import { removeBookAsync, appId } from '../redux/books/booksSlice';
 
-function BookList({ book }) {
+function BookList({ book, randomNum }) {
+  const [progress, setProgress] = useState(randomNum);
```

Comment:
- [ ] EXPECTED DESIGN

<img width="954" alt="1" src="https://github.com/MenendezON/bookstore/assets/77843050/66cfff1c-ecd9-411b-b43f-7092d41eb554">

- [ ] WHAT YOU DID

<img width="835" alt="2" src="https://github.com/MenendezON/bookstore/assets/77843050/9143e800-0004-4640-b621-bea80478bda7">

1. We have some key parts that are missing in the design as the progress bar percentage, 
2. The style of the progress bar should be identical to the one above on the original template
3. The user icon on the top-right is different from the one on the original design
4. The bookstore title is not well aligned as the above is showing 

<img width="386" alt="3" src="https://github.com/MenendezON/bookstore/assets/77843050/800aed9e-03f0-45f5-a05f-7969d2824777">

1. Make sure you fix this text brightness, I think your color is more blue than the one on the original design

<img width="322" alt="5" src="https://github.com/MenendezON/bookstore/assets/77843050/d53ef210-40e5-4138-b32a-1fd6f34c2764">

1. Please add a space around your update button as per the design above